pytest_splunk_addon/splunk.py

```diff
@@ -293,6 +299,26 @@ def splunk_search_util(splunk, splunk_setup, request):
 
     return search_util
 
+@pytest.fixture(scope="session")
+def suppress_internal_errors(request):
```

Comment:
For consistency I think function and parameter should be named similarly (choosing either ignore or suppress and stick to it)

Reply:
Sure will do the changes.